### DISCUSSION TITLE
Silly brain - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#22] [General]
# Silly brain - Alan Pope's blog

A re-enactment of an event yesterday evening.
I was just leaving an online game when I noticed a conversation among the Late Night Linux Telegram group about printing. One person quipped that people don’t print much anymore. Someone else suggested that they print more these days than they used to.
My brain saw this and thought “Huh, I wonder how many pages my printer has completed in its lifetime. I imagine that’s easy to find out.

https://popey.com/blog/2023/08/silly-brain/

**Comment by MariusQuabeck** [comment]
For my HP LaserJet MFP M28w, the lifetime page count is under Home --> Device configuration --> scroll down to Page Counts "Total Impressions".
I've printed 207 pages and scanned 71 since I got it in 2021. I assume I'll keep it for a decade by the rate I'm printing pages...

PS:
yes, that is ADHD (or at least the one that I'm diagnosed with is very similar), that's £5, you have my PayPal 😜

**Reply by popey** [reply]
Aha! 
![image](https://github.com/popey/popey.com-blog/assets/1841272/52faa894-ba10-49c5-aad2-d502d1674b94)

6736!

**Comment by MacSlow** [comment]
How come you can still use that printer without any issues of 'programmed-in obsolescence'? As much as I tried in the past, no printer I ever owned, made it past four years. The failures were always of the type 'just stopped working'. Never were there any mechanical failures. I only ever got printers from Canon or HP. While I kind of expected this sort of thing from Canon, I was surprised to see HP printers "gain" the same ability to just stop working over time.

BTW, did my barrage of DMs on Mastodon regarding 'Linux Matters' reach you?

Best regards...

Mirco
